packages/ui/src/elements/Table/OrderableTable.tsx

```diff
@@ -133,6 +133,14 @@ export const OrderableTable: React.FC<Props> = ({
           'Failed to reorder. This can happen if you reorder several rows too quickly. Please try again.',
         )
       }
+
+      if (response.status === 200 && (await response.json())['message'] === 'initial migration') {
+        throw new Error(
```

Comment:
This should be `throw new APIError`

Reply:
exported from `payload`, right? That gives an error. It's not intended to be used on the server?

Reply:
`import { APIError } from 'payload'`

This is all over our code base.

Reply:
Oh I didn't see that this was in `packages/ui`, disregard!